src/components/Header/Button/Profile/Profile.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import Profile from './Profile';
+
+describe('Profile', () => {
+  test('should contains a specific text', () => {

```

Comment:
The description needs to be more accurate.. In this case, it could be: `should display the loading text`